### DISCUSSION TITLE
OS X updates: Xcode 12.3 and Big Sur

### DISCUSSION THREAD

**Discussion by peterb** [Homebrew/discussions#321] [Everyday usage] [answered]
Is upgrading to the latest version of Xcode (12.3) and/or Big Sur alright or should they be avoided for now?

**Comment by ldanko** [comment, marked as answer]
After update to Xcode 12.3 I have to manually reinstall command line tools (`xcode-select --install`).

See https://github.com/Homebrew/discussions/discussions/310.
